proposals/NNNN-library-evolution.md

```diff
@@ -0,0 +1,369 @@
+# Library Evolution for Stable ABIs
+
+* Proposal: [SE-NNNN](NNNN-library-evolution.md)
+* Authors: [Jordan Rose](https://github.com/jrose-apple), [Ben Cohen](https://github.com/airspeedswift)
+* Review Manager: TBD
+* Status: **Awaiting review**
+* Implementation: Implemented in Swift 5 for standard library use
```

Comment:
Could you whip up an implementation PR that actually uses the command-line and attribute spelling you're proposing?

Reply:
PR here, but it's far easier to use the shipping toolchain to try it out with the underscores than one generated from that PR.

Reply:
PR where?

Reply:
Link added.

Reply:
Ah, I see it, thanks.